packages/create/package.json

```diff
@@ -40,13 +40,18 @@
         "@sap-ux/logger": "workspace:*",
         "@sap-ux/mockserver-config-writer": "workspace:*",
         "@sap-ux/project-access": "workspace:*",
+        "@sap-ux/ui5-config": "0.22.3",
         "chalk": "4.1.2",
         "commander": "9.4.0",
         "diff": "5.1.0",
+        "inquirer": "8.2.6",
```

Comment:
The `inquirer` dependency is a no-go here. The create module shall not depend on it.
Additionally, I don't think you need mem-fs and mem-fs-editor

Reply:
Fixed with commit: 496998c4003cd968bb4cf6346e9a2dd6bfd2bcd2